<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1600</width>
    <height>900</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Multi-Camera Collector</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
QWidget {
    font-family: "Segoe UI", "Arial", sans-serif;
    font-size: 10pt;
}

QMainWindow {
    background-color: #f0f0f0;
}

#main_container {
    background-color: #f0f0f0;
}

#left_panel, #right_panel {
    background-color: #f0f0f0;
    border: none;
}

#header {
    background-color: #ffffff;
    border-radius: 8px;
    border: 1px solid #dcdcdc;
}

#title_label {
    color: #212121;
    font-size: 16pt;
    font-weight: bold;
}

#status_label {
    color: #28a745;
    font-weight: bold;
}

#scrollArea, #scrollAreaWidgetContents {
    background-color: #ffffff;
    border-radius: 8px;
    border: 1px solid #dcdcdc;
}

#controls_container, #log_container {
    background-color: #ffffff;
    border-radius: 8px;
    border: 1px solid #dcdcdc;
}

QGroupBox {
    font-weight: bold;
    color: #333;
    border: 1px solid #dcdcdc;
    border-radius: 5px;
    margin-top: 1ex;
}

QGroupBox::title {
    subcontrol-origin: margin;
    subcontrol-position: top left;
    padding: 0 3px;
    background-color: #f0f0f0;
}

QLabel {
    color: #333;
}

QLineEdit, QComboBox, QSpinBox {
    border: 1px solid #dcdcdc;
    border-radius: 4px;
    padding: 5px;
    background-color: #ffffff;
}

QLineEdit:focus, QComboBox:focus, QSpinBox:focus {
    border-color: #0078d7;
}

QPushButton {
    background-color: #0078d7;
    color: white;
    border: none;
    border-radius: 4px;
    padding: 8px 16px;
    font-weight: bold;
}

QPushButton:hover {
    background-color: #005a9e;
}

QPushButton:pressed {
    background-color: #004578;
}

QCheckBox {
    spacing: 5px;
}

QTextEdit {
    border: 1px solid #dcdcdc;
    border-radius: 4px;
    background-color: #ffffff;
}
   </string>
  </property>
  <widget class="QWidget" name="main_container">
   <layout class="QHBoxLayout" name="horizontalLayout" stretch="3,1">
    <property name="spacing">
     <number>10</number>
    </property>
    <property name="leftMargin">
     <number>10</number>
    </property>
    <property name="topMargin">
     <number>10</number>
    </property>
    <property name="rightMargin">
     <number>10</number>
    </property>
    <property name="bottomMargin">
     <number>10</number>
    </property>
    <item>
     <widget class="QWidget" name="left_panel" native="true">
      <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,1">
       <property name="spacing">
        <number>10</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QScrollArea" name="scrollArea">
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>1175</width>
            <height>796</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <widget class="QWidget" name="camera_grid_container" native="true">
             <layout class="QGridLayout" name="camera_grid_layout">
              <property name="spacing">
               <number>10</number>
              </property>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="right_panel" native="true">
      <layout class="QVBoxLayout" name="verticalLayout_3" stretch="0,0,1">
       <property name="spacing">
        <number>10</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QWidget" name="controls_container" native="true">
         <layout class="QVBoxLayout" name="controls_layout"/>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="log_container" native="true">
         <layout class="QVBoxLayout" name="log_layout"/>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
